lib/manager/components/version/FeedVersionTabs.js

```diff
@@ -101,21 +106,37 @@ export default class FeedVersionTabs extends Component<Props> {
           unmountOnExit>
           <Tab eventKey={'feed'} title='Summary'>
             <Row>
-              {countFields.map(c => (
-                <Col xs={2} className='text-center' key={c}>
-                  <p
-                    title={`${version.validationSummary[c]}`}
-                    style={{marginBottom: '0px', fontSize: '200%'}}>
-                    {numeral(version.validationSummary[c]).format('0 a')}
-                  </p>
-                  <p style={{marginBottom: '0px'}}>{this.messages(c)}</p>
-                </Col>
-              ))}
+              {countFields.map(c => {
+                const summary = version.validationSummary
+                const diff = comparedVersion ? (summary[c] - comparedVersion.validationSummary[c]) : 0
+
+                return (
+                  <Col xs={2} className='text-center' key={c}>
+                    <div
+                      title={`${summary[c]}`} // FIXME: do we need the title prop?
+                      style={{marginBottom: '0px', fontSize: '200%'}}>
+                      {numeral(summary[c]).format('0a')}
+                      {comparedVersion && comparedVersionIndex && (
+                        <sup style={{width: 0, display: 'inline-block'}}>
+                          <BsLabel
+                            style={{fontSize: '50%', padding: '0 0.2em 0.1em'}}
+                            bsStyle={diff > 0 ? 'success' : (diff < 0 ? 'danger' : 'default')}
+                            title={`Difference compared to Version ${comparedVersionIndex || 'null'}`}>
+                            {formatDelta(diff)}
+                          </BsLabel>
+                        </sup>
+                      )}
+                    </div>
```

Comment:
Opportunity to make this its own component or add comments about what this is rendering. Otherwise, it's a bit hard to read.

Reply:
Good idea. Updated in https://github.com/ibi-group/datatools-ui/pull/528/commits/5fa63cd05c6aca8f7775f84b1a17791002cd6fac.